src/NewsletterBundle/Resources/config/services.xml

```diff
@@ -21,7 +21,7 @@
             <argument type="service" id="chameleon_system_newsletter.post_processor_collector" />
         </service>
 
-        <service id="chameleon_system_newsletter.user_data_factory" class="ChameleonSystem\NewsletterBundle\PostProcessing\Bridge\NewsletterUserDataFactory">
+        <service id="chameleon_system_newsletter.user_data_factory" class="ChameleonSystem\NewsletterBundle\PostProcessing\Bridge\NewsletterUserDataFactory" public="true">
         </service>
 
         <service id="chameleon_system_newsletter.mapper.signout_config" class="TPkgNewsletterMapper_PkgNewsletterModuleSingOutConfig">
```

Comment:
These 4 also need to be public.

Reply:
Comment will be obsolete with my pull request https://github.com/bestform/chameleon-base/pull/1